demo-server/templates/index-template.handlebars

```diff
@@ -59,24 +59,32 @@
             initWidget(authnWidget);
         }
 
+        function showResponse(text) {
+          // success callback function
+          var resultContainer = document.createElement('div');
+          resultContainer.classList.add('card');
+          resultContainer.style.overflowWrap = 'break-word';
+          resultContainer.style.marginTop = '20px';
+          resultContainer.style.fontFamily = 'monospace';
+          resultContainer.innerHTML = "";
+          resultContainer.appendChild(document.createTextNode(text));
+          document.getElementById('authnwidget').appendChild(resultContainer);
+        }
+
         /**
          * initialize the widget based on the operation mode
          */
         function initWidget(widget) {
           {{#if redirectless}}
           var config = {
-            // client_id: '', // redirectless requires this configuration
-            // response_type: 'token', // redirectless requires this configuration
+            client_id: 'ac_client', // redirectless requires this configuration
```

Comment:
Not sure if uncommenting/changing this was only for testing or a new default?

Reply:
Can't tell you how many times I told myself I needed to remember to switch this back